### DISCUSSION TITLE
Hackathon at AGU24 - who's in!?

### DISCUSSION THREAD

**Discussion by asteiker** [earthaccess-dev/earthaccess#775] [General]
@JessicaS11 proposed the idea of an unofficial sprint or hackathon at this year's AGU24 meeting.

- What day or time would work best? Could we aim for the Sunday before the meeting starts? If we decide to organize this, we need to get the word out quickly as folks may begin booking travel.
- What topic(s) or feature(s) would we like to focus on?
 -- @betolink shared ideas coming out of SciPy in #771 - we can cross-reference!

**Comment by JessicaS11** [comment]
For those of you who haven't seen, Pangeo is organizing a meeting the Saturday after AGU. [The organizers have encouraged us to submit earthaccess (and/or icepyx/hackweek topics) for their project sessions](https://discourse.pangeo.io/t/post-agu-pangeo-working-meeting-december-14-2024-in-washington-dc/4440/2). I'm tentatively planning to stay for this event and submit a topic (another good option could be xarray h5coro backend if @jpswinski or someone who knows more about what needs to be done is able to lead the charge). Thoughts on collaborating on this Saturday event versus a separate hack the Sunday before?

**Comment by jhkennedy** [comment]
I think it makes sense to combine it with the Pangeo meeting, and I'm planning to stay for the Pangeo meeting.

**Comment by betolink** [comment]
I like the Pangeo-earthaccess convo idea, maybe if it's not too much we can have an informal meetup during the week as I suspect some people interested on this will have to leave on Friday/Saturday and will miss it.

**Comment by JessicaS11** [comment]
Just registered for the Pangeo meeting on Saturday 14 Dec (and listed earthaccess as a project topic)!

> maybe if it's not too much we can have an informal meetup during the week as I suspect some people interested on this will have to leave on Friday/Saturday and will miss it.

Sounds like fun!

**Comment by itcarroll** [comment]
Also, registered. And mentioned earthaccess as well as anything to support cloud-optimizing NASA archives as topics.
